src/fetch-utils.ts

```diff
@@ -1,10 +1,11 @@
 import fetch from './fetch-cache';
 
 interface SourceConfigObject<U> {
+    allwaysUseFallback?: boolean;
```

Comment:
?

Reply:
500 feil skal håndteres forskjellig basert på endepunkt. 
I noen tilfeller vil vi vise en stor rød boks, men i andre tilfeller så skal appen fortsette å fungere som normalt. 

Så for å unngå stor-rød-boks når feature-appen gir 500 så setter vi den til å alltid bruke fallback.